packages/ag-grid-styles/src/agGridStyles/useAgGridStyles.ts

```diff
@@ -1,27 +1,10 @@
-import { makeStyles, createStyles, ClassNameMap } from '@equinor/fusion-react-styles';
+import { ClassNameMap, createStyles, makeStyles } from '@equinor/fusion-react-styles';
 import styles from './styles.jss.json';
 
-type agGridProps = {
-  indicateEditMode?: boolean;
-};
-
 export const useStyles = makeStyles(
-  (theme) =>
+  () =>
     createStyles({
       ...styles,
-      root: (props?: agGridProps) => ({
```

Comment:
i would leave root in there so it wont break for people using the root class, it could just be empty.